src/Microsoft.OpenApi.Hidi/OpenApiSpecVersionExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.OpenApi.Hidi
+{
+    public static class OpenApiSpecVersionExtension
+    {
+        public static OpenApiSpecVersion TryParseOpenApiSpecVersion(this string value)
```

Comment:
Extension methods on very commonly used types like string are not usually recommended.  If every use of a string did this it would hugely increase the surface are of the String class.  I would suggest just renaming this to OpenApiSpecVersionHelper and accept the string as a regular parameter.

Reply:
Thank you for pointing this out. Resolving...